lib/cmd/common/admin-storeitems.js

```diff
@@ -185,11 +185,11 @@ function update(guid, params, cb) {
 }
 
 function addIcon(guid, iconFile, cb) {
-  fhreq.uploadFile("/box/srv/1.1/admin/storeitem/uploadbinary", iconFile, {"type":"icon", "guid":guid}, "application/octet-stream" , cb);
+  fhreq.uploadFile("/box/srv/1.1/admin/storeitem/uploadbinary", iconFile, {"type":"icon", "guid":guid}, cb);
```

Comment:
you're removing the content type specifier.  That will mean that the request module will be setting the type, to what it thinks the data is, via the form-data module.
Is that a good idea? Or do you think it would be better to keep the specified content types and modify doRequestWithDataObjectValues() to add the specified content type to the form